### PR TITLE
Close button feature

### PR DESCRIPTION
**Pull request by jmparke**
Added a feature that allows a user to clear the previously selected suggestion.

### CODE REVIEW

**Comment by gavinr** [comment]
I merged our Travis fix ( #222) so the automated tests would run correctly - now that they're running correctly, looks like there are some linting errors: https://travis-ci.org/Esri/esri-leaflet-geocoder/builds/572742590?utm_source=github_status&utm_medium=notification

**Comment by jmparke** [comment]
Fixed lint errors

File: debug/sample.html [hunk 1]
```diff
@@ -28,36 +31,46 @@
...
+        attribution:
+          '&copy; <a href="https://osm.org/copyright">OpenStreetMap</a> contributors'
       }).addTo(map);
 
       var searchControl = L.esri.Geocoding.geosearch({
+        position: "topright",
+        collapseAfterResult: true,
+        expanded: false,
+        zoomToResult: true,
+        useMapBounds: true,
```
Comment (gavinr) [inline debug/sample.html]: Also, one small change - could you please review these extra properties you added to the sample page?

**Comment by 08Thug** [comment]
is the close feature is enabled!? is it working

**Comment by gavinr** [comment]
@08Thug this PR was not merged - see my comments above. A PR that includes these changes plus the fixes would be welcomed.

**Comment by gavinr** [comment]
Closing due to lack of activity. We would be open to a PR that adds similar functionality in the future. Thanks!

**Pull request closed without merging**